File: MATD74 - Algoritmos e Grafos/Atividades/Lista1/1015_DISTANCIAPONTOS.py
import math

a = input()
b = input()
ponto1 = a.split(' ')
ponto2 = b.split(' ')
ponto1[0], ponto1[1] = float(ponto1[0]), float(ponto1[1])
ponto2[0], ponto2[1] = float(ponto2[0]), float(ponto2[1])
difX = ponto1[0]-ponto2[0]
difY = ponto1[1]-ponto2[1]
distancia = math.sqrt(math.pow(difX, 2) + math.pow(difY, 2))

print('{0:.4f}'.format(distancia))

File: MATD74 - Algoritmos e Grafos/Atividades/Lista1/1015_DISTANCIAPONTOS.c
#include <stdio.h>
#include <math.h>

int main ()
{
  double ponto1[2], ponto2[2];
  scanf("%lf %lf", &ponto1[0], &ponto1[1]);
  scanf("%lf %lf", &ponto2[0], &ponto2[1]);
  double difX = ponto1[0]-ponto2[0];
  double difY = ponto1[1]-ponto2[1];
  double distancia = sqrt(pow(difX, 2.0) + pow(difY, 2.0));
  printf("%.4lf\n", distancia);
  return 0;
}
